### PR TITLE
Media Loader needs to check for techs in the registry

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Comment by pam** [comment]
**Tests passed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **CONFIRMED**

Commit: 63bd07ae2fd7196266bc7929d9062411156453ba
Build details: https://travis-ci.org/pam/video.js/builds/90207889

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: src/js/tech/loader.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /**
  * @file loader.js
  */
-import Component from '../component';
+import Component from '../component.js';
```
Comment (dmlap) [inline src/js/tech/loader.js]: Is `.js` the preferred syntax for doing this now?

**Comment by dmlap** [comment]
LGTM

File: src/js/tech/loader.js (hunk 1 above)
Comment (gkatsev) [inline src/js/tech/loader.js]: I always preferred being explicit about it.

**Pull request closed without merging**